packages/components-react/projects/react-ssr-wrapper/src/splitChildren.ts

```diff
@@ -0,0 +1,18 @@
+import type { ReactElement } from 'react';
+
+export const splitChildren = (
+  children: ReactElement | ReactElement[]
```

Comment:
This isn't right. Based on 
```ts
type PropsWithChildren<P = unknown> = P & { children?: ReactNode | undefined };
```
it should be `ReactNode | undefined` or even base it on `PropsWithChildren` via `Pick<PropsWithChildren<{}>, 'children'>`.